src/object/obj_class.c

```diff
@@ -109,7 +109,8 @@ daos_oclass_name2id(const char *name)
 
 	/* slow search path, it's for tool and not performance sensitive. */
 	for (oc = &daos_obj_classes[0]; oc->oc_id != OC_UNKNOWN; oc++) {
```

Comment:
@gnailzenh A further question is should we support more oclass name than defined?
Eg, if users pass RP_3G3, we should return id as: 3 << 24 | 3

And we need modify daos_oclass_id2name() to convert ID to oclass name(need be careful for overflow though)